sourcery_analytics/main.py

```diff
@@ -161,8 +163,7 @@ def cli_assess(
         console.print(f"[bold red]Found {count} errors.")
         raise typer.Exit(1)
 
-    console.print("[bold green]Assessment Complete")
-    console.print("[green]No issues found.")
+    console.print("[bold green]Assessment Complete", "[green]No issues found.")
```

Comment:
**suggestion (code_refinement):** Combine console print statements for cleaner output.

Merging these two messages into a single print statement is a good refinement for cleaner and more concise output. However, ensure that the intended formatting is preserved, as the current implementation might not separate the messages as expected.

```suggestion
    console.print("[bold green]Assessment Complete\n[green]No issues found.")
```